core/scripts/cre/environment/configs/workflow-don-tron.toml

```diff
@@ -42,11 +42,11 @@
   bootstrap_node_index = 0 # added for visibility, index of the node in the nodeset that will act as a bootstrap
   gateway_node_index = 0 # added for visibility, required when don_types include "gateway"
   env_vars = { CL_EVM_CMD = "" }
-  capabilities = ["ocr3", "custom-compute", "web-api-target", "web-api-trigger", "vault", "cron", "http-action", "http-trigger", "consensus"]
+  capabilities = ["ocr3", "custom-compute", "web-api-target", "web-api-trigger", "vault", "cron", "http-action", "http-trigger", "consensus", "don-time"]
```

Comment:
Is DON time a capability?

Reply:
honestly? I have no idea how to classify it. My understanding is that it is required to correctly use time in the workflows, but it is more of an internal capability than something users need to be aware of.